routes/account.js

```diff
@@ -1,20 +1,29 @@
 const express = require('express')
-const router =  express.Router()
+const router = express.Router()
 
 const { isAuthorised } = require('../middleware/auth')
 const { locals } = require('../utils/format')
 
 router.get('/index', isAuthorised, (req, res) => {
-    res.render('account/index', req.session?.user)
+    const payload = {
+        ...req.session?.user,
+        authorised: req.session?.authorised,
+    }
+    console.log('ppp', payload)
+    res.render('account/index', locals(payload))
```

Comment:
You should be able to access locals variables inside your ejs files without explicitly passing them in

If you create a middleware you can pass session data in

app.use((req, res, next) => {
  res.locals.authorized = req.session.authorized
next();
});


<%= authorized %>